releasenotes/notes/0.15/phase-estimation-circuit-c22d71d88873aa77.yaml

```
@@ -0,0 +1,16 @@
+---
+features:
+  - |
+    A new class :class:`~qiskit.circuit.library.PhaseEstimation` has been
+    added to :mod:`qiskit.circuit.library`. This circuit library class is
+    used for is used to estimate the phase :math:`\phi` of an eigenvalue
+    :math:`e^{2\pi i\phi}` of a unitary operator :math:`U`, provided with the
+    corresponding eigenstate :math:`|psi\rangle`. That is
```

We could be a bit more explicit that this is the circuit of one of the phase estimation algorithms
```suggestion
    added to :mod:`qiskit.circuit.library`. This circuit library class 
    is the circuit used in the original formulation of the phase estimation
    algorithm in [arXiv:quant-ph/9511026](https://arxiv.org/pdf/quant-ph/9511026.pdf).
    Phase estimation is the task to to estimate the phase :math:`\phi` of an eigenvalue
    :math:`e^{2\pi i\phi}` of a unitary operator :math:`U`, provided with the
    corresponding eigenstate :math:`|psi\rangle`. That is
```

---

Thanks, I'll add the link, but this is a markdown style link though and we need to use https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#external-links so I can't apply it directly

---

Done in: https://github.com/Qiskit/qiskit-terra/pull/4850/commits/3f9d9ac24413762954384f378063c1e5d2fad3d4